i18n/translations/en/my-forms.json

```diff
@@ -3,11 +3,14 @@
   "nav": {
     "drafts": "Drafts",
     "published": "Published",
-    "all": "All"
+    "all": "All",
+    "archived": "Archived"
   },
   "actions": {
     "resumeForm": "Resume editing form",
-    "createNewForm": "Create new form"
+    "createNewForm": "Create new form",
+    "unarchiveForm": "Restore form",
+    "duplicatePublishedForm": "Duplicate published form"
```

Comment:
Question: Why is the action here to duplicate (archived, published) rather than restore?

Reply:
Initial iteration was that they'd duplicate a published form instead of bringing it back. eg: Fresh and clean copy. Could be changed to whatever implementation we'd like.